modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -127,5 +126,44 @@ definition {
 			PRMMDFListing.viewOnList(campaignName = "Campaign Name");
 		}
 	}
+	@description = "This is a test for LPS-171637. Verify that an alert will appear when user don't fill a mandatory* field and that “Continue” button only will be clickable after fill all mandatory fields"
```

Comment:
Hi @MGabriellaS, could you please update the description to `"This is a test for LPS-171637. Verify that an alert will appear when user don't fill a mandatory field and that 'Continue' button only will be clickable after fill all mandatory fields"`?. I know that you pick up from Test Map, so it's my bad, sorry!

Reply:
No problem, I will change.